CPS-????/README.md

```diff
@@ -0,0 +1,83 @@
+---
+CPS: ?
+Title: Light Client Protocols
```

Comment:
@cleanerm5 if you could work with reviewers & editors to make or propose a title change, as per the above comment, it would be appreciated before the end-of-year work hiatus begins.

Reply:
It is about light client protocols, but not about the networking, it is about the technological properties of light client proofs to verify state transitions on Cardano in contracts on other chains.